tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -71,6 +72,58 @@ def test_fsx_lustre(
     _test_data_repository_task(remote_command_executor, mount_dir, bucket_name, fsx_fs_id, region)
 
 
+@pytest.mark.regions(["us-east-1"])
+@pytest.mark.instances(["c5.xlarge", "m6g.xlarge"])
+@pytest.mark.schedulers(["sge"])
+@pytest.mark.usefixtures("os", "instance", "scheduler")
+# FSx is not supported on CentOS 6
+@pytest.mark.skip_oss(["centos6"])
+# FSx is only supported on ARM instances for Ubuntu 18.04 and Amazon Linux 2
+@pytest.mark.skip_dimensions("*", "m6g.xlarge", "alinux", "*")
+@pytest.mark.skip_dimensions("*", "m6g.xlarge", "centos7", "*")
+@pytest.mark.skip_dimensions("*", "m6g.xlarge", "ubuntu1604", "*")
+def test_fsx_lustre_backup(
+    region, pcluster_config_reader, clusters_factory, test_datadir, os,
+):
+    """
+    Test all FSx Lustre related features.
+
+    Grouped all tests in a single function so that cluster can be reused for all of them.
+    """
+    mount_dir = "/fsx_mount_dir"
+    utc_now_plus_15 = datetime.datetime.utcnow() + datetime.timedelta(minutes=15)
+    daily_automatic_backup_start_time = utc_now_plus_15.strftime("%H:%M")
+    logging.info("daily_automatic_backup_start_time" + daily_automatic_backup_start_time)
+    cluster_config = pcluster_config_reader(
+        mount_dir=mount_dir, daily_automatic_backup_start_time=daily_automatic_backup_start_time
+    )
+    cluster = clusters_factory(cluster_config)
```

Comment:
Since there's a lot going on in this function, can you add comments before each cluster creation that explain what portion of the overall testing is fulfilled by it?